### PR TITLE
Statusmonitor r2008 latest

### PR DESCRIPTION
**Pull request by vijayrap**
request review, Test cases will be added in the next PR.

### CODE REVIEW

**Comment by vijayrap** [comment]
Request to check.

**Comment by kaweue** [comment]
recheck ha

**Review by lrebeja — APPROVED** [review]
As discussed UTs will be added later.
The rest looks good.

**Review by vvaitla-juni — APPROVED** [review]

File: deploy/1-create-operator.yaml [hunk 1]
```diff
@@ -633,24 +635,16 @@ spec:
                   - rbac
                   type: string
                 analyticsConfigAuditTTL:
-                  description: Time (in hours) the analytics config data entering the
```
Comment (michalskalski) [inline deploy/1-create-operator.yaml:636]: Please consider cherry-picking to your branch this commit https://github.com/Juniper/contrail-operator/pull/363/commits/8af3424f09d6207069a66272960b327b43313eb6 so in your change you will not remove analytics fields descriptions. I fixed that in master branch, did not know that 2008 branch is still active.

Comment (vijayrap) [reply deploy/1-create-operator.yaml:636]: Will take care of it, Thank you for notifying.

**Pull request merged**